src/Model/Widget/Line.php

```diff
@@ -58,6 +59,13 @@ public static function fromString(string $string): self
         ], null);
     }
 
+    public static function parse(string $string): self
+    {
+        return self::fromSpans(
+            SpanParser::new()->parse($string)
```

Comment:
not related to this PR but I want to make all `fromFoos(` variadics, so should be `fromSpans(...SpanParser::new()->parse($string))`

Reply:
I love this idea!

Reply:
it should already be the case, but I obviously missed some :smile: